### ISSUE TITLE
Change HTML Location for Users 

### ISSUE THREAD

**Issue by it-bud** [it-bud/frontend-project#3]
Change the location of the user in the HTML file. This should be easy to do. Make sure to comment on the issue here if you think to take the issue. Thank you!

**Comment by Saurav-Sutaria** [comment]
Hello, I am beginner in this issue and want to contribute but don't know how to get started. Will you help how to get started?

**Comment by it-bud** [comment]
Absolutely. Thank you for commenting. I made this repository to help learners to contribute to the open source projects. 

I will be upload a detailed instruction on the README file. In this project, you will learn
1.  How to clone the project 
2. How to create a new branch 
3. How to pull the main branch 
4. How to make some changes 
5. How to use git commands like git add , git status and git log
6. How to commit and why to commit
7. How to push and create  a pull request 
8. And so on. 

Thank you.

**Comment by it-bud** [comment]
> 
> 
> Hello, I am beginner in this issue and want to contribute but don't know how to get started. Will you help how to get started?

Hi Saurav, I saw that you were able to contribute to the project. Thank you for that. Would you help me improve the contributing page for the projects? All the steps that you followed. You can check the issue in Issues. This will greatly help other people wanting to learn how to contribute.Thank you. I appreciate your collaboration.

**Comment by Saurav-Sutaria** [comment]
Sure,
Means I have to improve the contributing.md file?

**Comment by it-bud** [comment]
Yes Saurav. Since you will work on documentation, please do git checkout -b documentation/contribution. Lets follow easy to use branching guidelines. I will add you in the contributors list.

**Comment by Saurav-Sutaria** [comment]
Can you please explain me something about this issue?

**Comment by it-bud** [comment]
@all-contributors please add @Saurav-Sutaria  for documentation

**Comment by it-bud** [comment]
> 
> 
> Can you please explain me something about this issue?

Absolutely. There is a contributing.md page in the main branch. 
Make sure to do ` git pull origin main ` to update the working repository
After that, create a new branch called documentation/contributing 
In the Contributing,MD file, please write the documentation on how to clone , pick up issue, make changes , commit and push in the repository which will help other people to understand how to contribute. This will be written in Markdown Format, you can find more details about Markdown Syntax in https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax

**Comment by it-bud** [comment]
@@all-contributors please add @t-rhex for documentation

**Comment by it-bud** [comment]
@all-contributors please add @saurabh853  for documentation
